### ISSUE TITLE
The plug-in in the display panel sometimes does not shrink as the display panel compresses.

### ISSUE THREAD

**Issue by ysl-design** [ros-visualization/rviz#1758]
Describe your issue here and explain how to reproduce it.

### Your environment
* OS Version: Ubuntu 20.04
* ROS Distro: Noetic
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
    ```
   [ INFO] [1658890616.733419800]: rviz version 1.14.14
   [ INFO] [1658890616.733471700]: compiled against Qt version 5.12.8
   [ INFO] [1658890616.733501700]: compiled against OGRE version 1.9.0 (Ghadamon)
   [ INFO] [1658890616.741053500]: Forcing OpenGl version 0.
   [ INFO] [1658890616.894986100]: Stereo is NOT SUPPORTED
   [ INFO] [1658890616.895090400]: OpenGL device: llvmpipe (LLVM 12.0.0, 256 bits)
   [ INFO] [1658890616.895148600]: OpenGl version: 3.1 (GLSL 1.4).
    ```
* System locale, zh_CN.UTF-8

When I stretch the displays panel first, then compress it, and then double-click the plug-in component in the displays panel (such as the drop-down list), the picture below appears. It seems that the components don't adapt to size as the displays panel compresses. I do not know whether the problem is caused by the QT or the displays panel.
![image](https://user-images.githubusercontent.com/67178569/181165726-22afaa22-f9cc-4325-b73a-9fce90f99336.png)

**Comment by rhaschke** [comment]
I cannot reproduce this issue. Could you please provide more detailed instructions and a video as well?

https://user-images.githubusercontent.com/5376030/181205794-3ee24b7f-cc94-46ae-82f4-a135c89b42a6.mp4

**Comment by ysl-design** [comment]
Sorry, I can't upload the video right now. The procedure is as follows:
### step1:
![image](https://user-images.githubusercontent.com/67178569/181242548-47466a86-5f09-4fe1-b6a5-a073f1e19c38.png)
### step2:
![image](https://user-images.githubusercontent.com/67178569/181242716-b702020e-01a1-4361-8bd2-d03727746cd3.png)
### step3:
![image](https://user-images.githubusercontent.com/67178569/181242840-855760c0-01ea-4c95-9a7b-82d1f88a5b4b.png)
### result:
![image](https://user-images.githubusercontent.com/67178569/181243185-bef34257-3be0-40be-a17b-7a5a6acfccc4.png)

**Comment by rhaschke** [comment]
Thanks for the clarification. Now, I can reproduce the issue.

**Comment by rhaschke** [comment]
Could you please test https://github.com/ros-visualization/rviz/pull/1760?

**Comment by ysl-design** [comment]
I tried #1760  and the problem did not recur.

**Comment by rhaschke** [comment]
Closed via #1760.
